spec/rubocop/cop/rspec/repeated_description_spec.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+describe RuboCop::Cop::RSpec::RepeatedDescription do
+  subject(:cop) { described_class.new }
+
+  it 'registers an offense for repeated descriptions' do
```

Comment:
Yeah good catch. I actually thought I had more than one test here but I guess not. I'll add a few more